<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>COVID-19 vaccines:</title>
    <meta charset="utf-8" />
    <meta name="author" content="David Benkeser, PhD MPH   Emory University  Department of Biostatistics and Bioinformatics " />
    <script src="index_files/header-attrs-2.6/header-attrs.js"></script>
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# COVID-19 vaccines:
## Correlates and sieve analysis
### David Benkeser, PhD MPH<br> <span style="font-size: 75%;"> Emory University<br> Department of Biostatistics and Bioinformatics </span>
### <svg viewBox="0 0 512 512" style="height:1em;fill:currentColor;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"> <path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"></path></svg> <a href = "https://twitter.com/biosbenk" style = "color: white;"><span class="citation">@biosbenk</span></a> <br> <svg viewBox="0 0 512 512" style="height:1em;fill:currentColor;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"> <path d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"></path></svg> <a href="https://bit.ly/corrsieve" style="color: white;">bit.ly/corrsieve</a> <br>

---



class: inverse, center, middle

.huge[Vaccine correlates]



&lt;style type="text/css"&gt;
.remark-slide-content {
    font-size: 22px
}
&lt;/style&gt;

---

## Correlates of risk/protection

Two, interrelated goals of correlates analysis are to 

* identify/validate possible __surrogate endpoints__;
* understand __protective mechanisms__ of vaccines. 

If an __immune correlate__ is established to __reliably predict vaccine efficacy__, then subsequent efficacy trials may use the CoP as the __primary endpoint__. 

__Accelerates approval__ of

* existing vaccines in __different populations__ (e.g., children);
* __new vaccines__ in the same class.

---

## Correlates of risk/protection

.large[__Two levels of correlates analysis__\*:]

__Correlates of risk:__

* Correlation of immune response in vaccine recipients with outcome
* Risk prediction
* Evaluates associative parameters

__Correlates of protection__

* Evaluate immune response's ability to predict vaccine efficacy
* Evaluates causal parameters

&lt;br&gt; &lt;br&gt; &lt;br&gt; 

.small[\* [Plotkin and Gilbert (2012)](https://doi.org/10.1093/cid/cis238), [Qin et al (2007)](https://doi.org/10.1086/522428)]

---

## Correlates of risk

&lt;img src="img/hr_forest_bindRBD_mock.png" height="250px" style="display: block; margin: auto;" /&gt;

* (Stratified) __Cox model__ in vaccinated participants
* Adjusted hazard ratios per 10-fold difference in marker

---

## Correlates of risk

&lt;img src="img/marginalized_risks2_mock.png" height="400px" style="display: block; margin: auto;" /&gt;

* Population-level __cumulative incidence__ as a function of immune marker.

---

## Correlates of risk

.pull-left[

&lt;img src="img/PLOT_Day57bindRBD_pointwiseCI.png" height="250px" style="display: block; margin: auto;" /&gt;

* Nonparametric __threshold finding__ approach.

]

.pull-right[

&lt;img src="img/cor2.png" height="250px" /&gt;

* __Machine learning prediction__ using different sets of immune responses.\*

]

&lt;br&gt; &lt;br&gt;
.small[\* [Neidich et al (2019)](https://doi.org/10.1172/JCI126391)]

---

## Correlates of protection

__Effect modifiers of VE__

* How/does VE vary across subgroups defined by immune response?
* E.g., [Juraska et al (2020)](https://doi.org/10.1093/biostatistics/kxy074)


__Mediators of VE__

* What percentage of VE is attributable to immune response?
* [Cowling et al (2019)](https://doi.org/10.1093/cid/ciy759)
* [Benkeser et al (2021+)](https://arxiv.org/abs/2103.02643)

__Stochastic interventional VE__

* How/would shifting the immune response distribution impact VE?
* [Hejazi et al (2020)](https://doi.org/10.1111/biom.13375)

---

## Measuring correlates

Running assays on &gt;30k samples is .red[expensive] and __statistically unnecessary__.

Instead we use a __case-cohort design__ ([Prentice, 1986](https://www.jstor.org/stable/2336266?seq=1)) to __measure immune responses__ in 
* a stratified random subcohort (\~1600 individuals)
* all SARS-CoV-2 endpoints

&lt;img src="img/casecohort.png" height="250px" style="display: block; margin: auto;" /&gt;

---

## Statistical challenges

__Estimation in two-phase designs__

* Individuals who contract COVID may __differ from other participants__.
* Two-phase design .red[over-samples] these individuals.
* Augmented/inverse weighting approaches to __account for differences__.

__Low case numbers due to highly effective vaccines__

* Power for CoP analysis driven by __vaccine breakthroughs__.
* CoR `\(&gt;\)` 25 breakthroughs; CoP `\(&gt;\)` 50 breakthroughs.

__Remote computing__

* Moderna's data-sharing agreement with USG gives exclusive control to Moderna until the summer.
* Our correlates code must be run on their machines.

---

## Transparency and reproducibility

__CoVPN statisticians are committed to open science.__

* An in-progress, version-controlled SAP is [available](https://figshare.com/articles/online_resource/CoVPN_COVID-19_Vaccine_Efficacy_Trial_Immune_Correlates_SAP/13198595) for review.
* An [open-source GitHub repository](https://github.com/CoVPN/correlates_reporting) is being developed to implement methods involved in the SAP. 
  - Tools: R Markdown, `Make`, `renv`, `bookdown`, Travis CI, AWS

All primary analyses are undergoing double coding and code verification.

* Proof-of-concept for validation of ML in regulatory context.

&lt;br&gt;

Moderna correlates analysis will occur .red[early April] ðŸ˜±.

---

class: inverse, center, middle

.huge[Sieve analysis]

---

## Sieve Analysis

&lt;img src="img/sieveConcept.png" height="500px" style="display: block; margin: auto;" /&gt;

---

## Sieve Analysis

&lt;img src="img/sieveConcept2.png" height="500px" style="display: block; margin: auto;" /&gt;

---

## Sieve Analysis

&lt;img src="img/sieveConcept3.png" height="500px" style="display: block; margin: auto;" /&gt;
---

## HIV vaccines -- RV144 trial

The __ALVAC/AIDSVAX vaccine__ was evaluated in a Phase III efficacy trial. 

* Antigen consists of three HIV-1 envelope protein sequences.
* Efficacy evaluated in 16,395 HIV negative participants in Thailand. 

&lt;img src="img/rv144Figure.png" height="300px" style="display: block; margin: auto;" /&gt;

---

## Quantifying sieve effects

Data used for __vaccine sieve analysis__: 

* Random vaccine ( `\(V=1\)` ) or control ( `\(V=0\)` ) assignment.
* Infection/disease status ( `\(Y=1\)` or `\(Y=0\)` ) by end of followup. 
* Genotype of infecting pathogen classified into strains of interest, e.g., matched to vaccine ( `\(J=1\)` ) or mismatched to vaccine ( `\(J=2\)` ).

What summary of these data quantifies whether and __how vaccine efficacy differs__ based on the __genotype__ of the pathogen?

---

## Quantifying sieve effects

&lt;img src="img/sieveConcept2.png" height="500px" style="display: block; margin: auto;" /&gt;

---

## Quantifying sieve effects

__Idea:__ Proportion of vaccine-matched endpoints in vaccine vs. control: $$
P(J = 1 \ | \ V = 1, Y = 1) \ \text{vs.} \ P(J = 1 \ | \ V = 0, Y = 1)  \ . 
$$

If a __sieve effect__ is present, these proportions should be .red[different].

&lt;img src="img/rtssIdea1.png" height="350px" style="display: block; margin: auto;" /&gt;

---

## Quantifying sieve effects

.red[Problem:] Only looking at the __population with the endpoint__! 

Population with endpoint likely differs population enrolled in trial! 

* No __causal interpretation__ of this quantity.

How can we __quantify sieve effects__ with respect to the __causal effect__ of the vaccine? 

---

## Quantifying sieve effects

Idea: __VE__ compares proportion with __any endpoint__ in __vaccine vs. control__. $$
P(Y = 1 \ | \ V = 1) \ \mbox{vs.} \ P(Y = 1 \ | \ V = 0) \ .
$$

What about studying __strain-specific efficacy__, defined as a comparison of the proportion of __strain-specific endpoints__ in __vaccine vs. control__. $$
P(Y = 1, J = j \ | \ V = 1) \ \mbox{vs.} \ P(Y = 1, J = j \ | \ V = 0) \ \ \mbox{for `\(j=1,2\)`.}
$$

Under assumptions, this is the __average causal effect__ of the vaccine on the incidence of __strain-specific endpoints__.

---

## Quantifying sieve effects

Typically we express `\(j\)`-specific __vaccine efficacy__ as $$
VE_j = 1 - \frac{P(Y = 1, J = j \ | \ V = 1)}{P(Y = 1, J = j \ | \ V = 0)}
$$
and we test the sieve null hypothesis $$
H_0: VE_1 = VE_2 \ . 
$$

Is the multiplicative effect of the vaccine the same against `\(J=1\)` endpoints as it is against `\(J=2\)` endpoints?

---

## Estimating sieve effects

We have developed methods that __estimate and test sieve effects__ using flexible regression techniques. 

* control for informative censoring
* control for informative vaccine uptake
* nice efficiency properties

[Benkeser et al (2018)](https://www.tandfonline.com/doi/abs/10.1080/01621459.2018.1529594), [Benkeser et al (2017)](https://onlinelibrary.wiley.com/doi/10.1002/sim.7337), [Benkeser et al (2020)](http://journal-sfds.fr/article/view/786)

---

## Sieve results

The RV144 vaccine exhibited __sieve effects at amino acid position 169__ of the V2 loop of the HIV Envelope protein 


&lt;img src="img/rv144.png" height="400px" style="display: block; margin: auto;" /&gt;

---

## Sieve analysis for COVID 

Great interest in __sieve analysis__ for SARS-CoV2 .red[variants of concern].

* We are planning a sieve analysis of __Moderna's Phase 3__ trial.
* Other companies may come around as well...
* Competing interests? Maybe not this time.

---

## Current work

User-friendly approach to assessing __effect modification of sieve effects__. 

* E.g., is the strength of the sieve effect modified by age?
* Joint with __Guandong (Elliot) Yang and Laura Balzer (UMass)__.

Accounting for __missing genotypes__ in sieve analysis.

* Only \~50% of Moderna cases have genetic sequences.
* Appropriately accounting for missing data.
* Joint work with __Ziyue (Jeremy) Wu (Emory)__.

---

## Amazing statisticians

.pull-left[.tiny[
__Leadership__
* Dean Follmann (NIAID)
* Yonghong Gao (BARDA)
* Peter Gilbert (FHCRC, UW)

__NIAID__
* Martha Nason
* Mike Fay
* Pretty much all of NIAID Biostatistics

__CoVPN__
* Alex Luedtke (UW)
* Marco Carone (UW)
* IvÃ¡n DÃ­az (Weill-Cornell)
* Nima Hejazi (Berkeley)
* many others!

]]

.pull-right[.tiny[
__Fred Hutch__
* Holly Janes
* Youyi Fong
* Yunda Huang
* Michal Juraska
* Ying Huang
* Ollivier Hyrien
* many others!

__OWS Company statisticians__
* Too many to name!
]
]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create();
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
